heron/shell/src/python/handlers/killexecutorhandler.py

```diff
@@ -20,10 +20,13 @@
 
 
 ''' killexecutorhandler.py '''
+from future.standard_library import install_aliases
+install_aliases()
+
 import logging
 import os
 import signal
-import urlparse
+from urllib.parse import parse_qsl
```

Comment:
Looks like this is a package only valid in py3?

Reply:
it's available in python2 with the first two lines import of this file

```
from future.standard_library import install_aliases
install_aliases()
```